ros_ws/src/navigation_stack/car_cartographer/launch/cartographer_offline_fast.launch

```diff
@@ -0,0 +1,41 @@
+<!--
+  Copyright 2016 The Cartographer Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<!-- Modified by the F1/10 Autonomous Racing Project Group -->
+
+<launch>
+  <param name="/use_sim_time" value="true" />
+
+  <arg name="ros_version" default="kinetic"/>
+
+  <group if="$(eval ros_version=='kinetic')">
+    <node name="cartographer_offline_node" pkg="cartographer_ros" type="cartographer_offline_node" args="
+          -configuration_directory $(find car_cartographer)/config
+          -configuration_basename cartographer_config_kinetic.lua
+          -urdf_filename $(find car_cartographer)/files/racer.urdf
+          -bag_filenames $(arg bag_filenames)" output="screen" required="true">
```

Comment:
The argument `bag_filenames` is used here but not declared in the launch file. It seems to work, but I'm not sure if it's supposed to. Do you think the argument should be declared in the launch file?

Reply:
Arguments don't have to be declared with an `<arg>` tag if you don't need to set a default value. I admit it would help with readability though, so I'll add it.